### ISSUE TITLE
Latest Docker image (0.10.5) has broken PYTHONPATH

### ISSUE THREAD

**Issue by xiang-zhu** [localstack/localstack#1753]
Hi,

We just noticed in the latest docker image, the PATHONPATH is not set correctly.

From https://github.com/localstack/localstack/blob/v0.10.5/Dockerfile#L55 it's pointing to `/opt/code/localstack/.venv/lib/python3.6/site-packages` but the latest image has updated to Python 3.7, so the path instead should be `/opt/code/localstack/.venv/lib/python3.7/site-packages`

**Comment by imriss** [comment]
This could overwrite the `PATHONPATH` value:
```
DOCKER_FLAGS='-e PYTHONPATH=/opt/code/localstack/.venv/lib/python3.7/site-packages' localstack start --docker; 
```

**Comment by whummer** [comment]
Thanks for reporting @xiang-zhu @imriss - should be fixed in #1756
